packages/office-ui-fabric-react/src/components/MessageBar/examples/MessageBar.Basic.Example.tsx

```diff
@@ -1,160 +1,184 @@
+        Visit our website.
+      </Link>
+    </MessageBar>
+  );
 
-    <StackItem>
-      <Label>Warning MessageBar - single line, with dismiss and action buttons</Label>
-      <MessageBar
-        messageBarType={MessageBarType.warning}
-        isMultiline={false}
-        onDismiss={log('test')}
-        dismissButtonAriaLabel="Close"
-        actions={
-          <div>
-            <MessageBarButton>Action</MessageBarButton>
-          </div>
-        }
-      >
-        Warning lorem ipsum dolor sit amet, a elit sem interdum consectetur adipiscing elit.{' '}
-        <Link href="www.bing.com" target="_blank">
-          Visit our website.
-        </Link>
-      </MessageBar>
-    </StackItem>
+  const WarningExample = () => (
+    <MessageBar
+      messageBarType={MessageBarType.warning}
+      isMultiline={false}
+      onDismiss={resetChoice}
+      dismissButtonAriaLabel="Close"
+      actions={
+        <div>
+          <MessageBarButton>Action</MessageBarButton>
+        </div>
+      }
+    >
+      Warning MessageBar content.
+      <Link href="www.bing.com" target="_blank">
+        Visit our website.
+      </Link>
+    </MessageBar>
+  );
 
-    <StackItem>
-      <Label>Warning MessageBar - defaults to multiline, with dismiss and action buttons</Label>
-      <MessageBar
-        onDismiss={log('test')}
-        dismissButtonAriaLabel="Close"
-        messageBarType={MessageBarType.warning}
-        ariaLabel="Aria help text here"
-        actions={
-          <div>
-            <MessageBarButton>Yes</MessageBarButton>
-            <MessageBarButton>No</MessageBarButton>
-          </div>
-        }
-      >
-        Warning lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi luctus, purus a lobortis tristique, odio augue pharetra
-        metus, ac placerat nunc mi nec dui. Vestibulum aliquam et nunc semper scelerisque. Curabitur vitae orci nec quam condimentum
-        porttitor et sed lacus. Vivamus ac efficitur leo. Cras faucibus mauris libero, ac placerat erat euismod et. Donec pulvinar commodo
-        odio sit amet faucibus. In hac habitasse platea dictumst. Duis eu ante commodo, condimentum nibh pellentesque, laoreet enim. Fusce
-        massa lorem, ultrices eu mi a, fermentum suscipit magna. Integer porta purus pulvinar, hendrerit felis eget, condimentum mauris.
-        <Link href="www.bing.com" target="_blank">
-          Visit our website.
-        </Link>
-      </MessageBar>
-    </StackItem>
+  const WarningExample2 = () => (
+    <MessageBar
+      onDismiss={resetChoice}
+      dismissButtonAriaLabel="Close"
+      messageBarType={MessageBarType.warning}
+      ariaLabel="Aria help text here"
+      actions={
+        <div>
+          <MessageBarButton>Yes</MessageBarButton>
+          <MessageBarButton>No</MessageBarButton>
+        </div>
+      }
+    >
+      Warning lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi luctus, purus a lobortis tristique, odio augue pharetra metus,
```

Comment:
Maybe add back something about this example allowing overflowing to multiple lines, since it omits the `isMultiline={false}`? (we will most definitely want to rethink some of these option names/behavior when rewriting...)